renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "automerge": true,
```

Comment:
@nLight Are you okay with this being auto-merge? I understood that we didn't want to go with it so far but this project seems small enough that we can test it with this feature

Reply:
Forget my comment, just saw that you have added it 👍

Reply:
🤘